src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,21 @@
-export const TodoFilter = () => (
+type Props = {
+  filter: {
+    query: string;
+    sort: string;
+  },
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  setFilter: (arg: any) => void;
```

Comment:
Don't disable the eslint, pass the correct type for this function. 
`(arg: any) => void;` is not the best choice